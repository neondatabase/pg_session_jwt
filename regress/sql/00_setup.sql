-- This file contains the setup for pg_session_jwt tests
-- Set the JWK parameter as a GUC parameter before connection start
\set jwk '{"kty":"OKP","crv":"Ed25519","x":"PLACEHOLDER_KEY"}'
ALTER SYSTEM SET pg_session_jwt.jwk = :'jwk';
SELECT pg_reload_conf();

CREATE SCHEMA IF NOT EXISTS auth;
CREATE EXTENSION pg_session_jwt;
