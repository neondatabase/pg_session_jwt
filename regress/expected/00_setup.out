-- This file contains the setup for pg_session_jwt tests
-- Set the JWK parameter as a GUC parameter before connection start
\set jwk '{"kty":"OKP","crv":"Ed25519","x":"PLACEHOLDER_KEY"}'
ALTER SYSTEM SET pg_session_jwt.jwk = :'jwk';
ALTER SYSTEM
SELECT pg_reload_conf();
 pg_reload_conf 
---------------
 t
(1 row)

CREATE SCHEMA IF NOT EXISTS auth;
CREATE SCHEMA
CREATE EXTENSION pg_session_jwt;
CREATE EXTENSION
